src/component.js

```diff
@@ -67,6 +67,12 @@ function _isPayPalDomain() : boolean {
   return window.mockDomain === 'mock://www.paypal.com' || isPayPalDomain();
 }
 
+function _isShoppingSdkComponent() : boolean {
+  const componentParam = (getSDKQueryParam('components') || '').toLowerCase();
```

Comment:
I'd recommend using `getComponents()` and doing an `indexOf()` check rather than just checking if the entire param === 'shopping'.

https://github.com/paypal/paypal-sdk-client/blob/6b4fe0dd9d31f4176980a007a908864328f834b0/src/global.js#L93

Can we also use the `COMPONENTS` enum here rather than hard-coding?

Reply:
Thanks, I'll update the PR